### ISSUE TITLE
Everything breaks when using commands with no arguments

### ISSUE THREAD

**Issue by AlignedCookie88** [VihnCraft/VCommandParser#3]
When running commands with no arguments, you are given the error message "You are missing some arguments".

Tab completing commands with no arguments reports an internal exception.

**Comment by AlignedCookie88** [comment]
```
Caused by: java.lang.ArrayIndexOutOfBoundsException: Index 0 out of bounds for length 0
	at com.vihncraft.libs.vcommandparser.argparser.ArgParser.parseNext(ArgParser.java:68) ~[VCommandParser-1.0.0.jar:?]
	at com.vihncraft.libs.vcommandparser.argparser.ArgParser.autoComplete(ArgParser.java:94) ~[VCommandParser-1.0.0.jar:?]
	at com.vihncraft.libs.vcommandparser.InternalTabCompleter.onTabComplete(InternalTabCompleter.java:28) ~[VCommandParser-1.0.0.jar:?]
	at org.bukkit.command.PluginCommand.tabComplete(PluginCommand.java:138) ~[paper-api-1.20.1-R0.1-SNAPSHOT.jar:?]
	... 22 more
```

**Comment by AlignedCookie88** [comment]
Fixed in 6cc270b
